content/docs/what-is-kudo.md

```diff
@@ -0,0 +1,133 @@
+# What is KUDO
+
+[[toc]]
+
+## Overview
+
+Kubernetes Universal Declarative Operator (KUDO) provides a declarative approach to building production-grade Kubernetes [operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/). To quote the official documentation: _"Operators are software extensions to Kubernetes that make use of custom resources to manage applications and their components"_. While Kubernetes already comes with a lot of built-in automation to run simple workloads, complex scenarios often need a human operator. This is where KUDO is designed to help.
+
+## Motivation
+
+Building Kubernetes operators is not easy. While existing tools and libraries like [kubebuilder](https://book.kubebuilder.io/) and [Operator Framework](https://github.com/operator-framework/getting-started) and Kubernetes [control loop](https://kubernetes.io/docs/concepts/#kubernetes-control-plane) help with some aspects, developing an operator is still quite an undertaking. This is where the KUDO approach differs. It utilizes a Universal and Declarative approach (the UD in KUDO), doesn't require any code and significantly accelerates the development of operators. In a nutshell, KUDO does all the "Kubernetes heavy lifting" allowing the developers to concentrate on the task at hand.
+
+## When would you use KUDO
```

Comment:
Maybe it makes sense to link to the comparison page from here?

Reply:
It does! Will add a link